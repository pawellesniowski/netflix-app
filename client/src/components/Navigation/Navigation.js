import React from 'react';

import './Navigation.scss';
import Button from "../ui-elements/Button/Button";

const Navigation = () => {
  return (
    <nav className="navigation" data-cy="navigation">
      <Button message="New search"/>
    </nav>
  )
};

export default Navigation;
