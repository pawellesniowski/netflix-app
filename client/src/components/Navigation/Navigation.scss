@import "../../common/styles/colors.scss";
@import "../../common/styles/mixins.scss";

.navigation {
  @include contentContainer;
  display: flex;
  justify-content: flex-end;
  background-color: $color-grey-black;
}
