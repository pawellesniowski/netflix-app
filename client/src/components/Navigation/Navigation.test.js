import React from 'react';
import {shallow} from 'enzyme';
import Navigation from "./Navigation";

describe("Navigation Component", () => {
    let component;
    beforeEach(() => {
        component = shallow(<Navigation/>);
    });

    it("Renders the container", () => {
       expect(component.find('[data-cy="navigation"]')).toHaveLength(1);
    });

    it("Renders children elements", () => {
        expect(component.find('Button')).toHaveLength(1);
    });

    it("Renders Navigation Component", () => {
        expect(component).toMatchSnapshot();
    });
});


