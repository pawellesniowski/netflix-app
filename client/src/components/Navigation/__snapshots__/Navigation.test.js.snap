// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component Renders Navigation Component 1`] = `ShallowWrapper {}`;
